src/libsync/bandwidthmanager.cpp

```diff
@@ -280,7 +281,7 @@ void BandwidthManager::relativeDownloadMeasuringTimerExpired()
     _relativeDownloadDelayTimer.setInterval(realWaitTimeMsec);
     _relativeDownloadDelayTimer.start();
 
-    int jobCount = _downloadJobList.count();
+    size_t jobCount = _downloadJobList.size();
```

Comment:
Same here with `auto jobCount`

Reply:
I had thought about this, but since the file was not using auto anywhere went with size_t. You are right. It is almost always auto. Done with d6f9bd00174af3eb7ee05ac72a7e3695a351a548.

Reply:
Yeah I know what you mean ^^ Being tempted to over-optimize stuff when you just want to focus on the actual improvement and I also had to find a balance ;-) Thanks for taking the time :)